ted2zim/scraper.py

```diff
@@ -700,7 +687,7 @@ def create_zim(static_folder, zim_path, title, description, name):
         "zim": zim_path,
         "scraper": "Ted scraper : https://github.com/openzim/ted",
         "source": "https://new.ted.com",
-        "tags": "_category:ted;ted",
+        "tags": "_topic:ted;ted",
```

Comment:
here this is different. `_category` is a special tag that we use to categorize ZIM files (see the list of folders on https://download.kiwix.org/zim/

Reply:
Thanks for pointing that out